New evaluation.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>New Evaluation</title>
+    
+    
+<link rel="stylesheet" href="css/New%20Evaluation.css">
+</head>
+<body>
+    
+    <form id="candidate">
+        <input type="text" placeholder="Candidate" name="Candidate">
+        <input type="text" placeholder="Interviewer" name="Interviewer">
+        <input type="date">
+    </form>
+    
+    <h1>Techinical Level</h1>
+<form>
+    <table align="centre">
```

Comment:
Hmm... interesting value for the `align` attribute... What are the valid values it can hold, by the way? Also, would there be any other, potentially better, places to put our styling?